test/spriter.test.js

```diff
@@ -0,0 +1,97 @@
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+const { Buffer } = require('buffer');
+const File = require('vinyl');
+const sinon = require('sinon');
+const SVGSpriter = require('../lib/svg-sprite.js');
+
+describe('testing SVGSpriter', () => {
+    let spriter;
+    const TEST_SVG = 'fixture/svg/single/weather-clear.svg';
+    const TEST_EMPTY_SVG = '<svg></svg>';
+
+    before(() => {
```

Comment:
Just to confirm, this is intentional? Like, we only need it to run once?

Reply:
No. That's a mistake, I'll fix it